functorch/_src/vmap.py

```diff
@@ -226,6 +230,11 @@ def vmap(func: Callable, in_dims: in_dims_t = 0, out_dims: out_dims_t = 0) -> Ca
         out_dims (int or Tuple[int]): Specifies where the mapped dimension
             should appear in the outputs. If :attr:`out_dims` is a Tuple, then
             it should have one element per output. Default: 0.
+        use_batched_random (str): Specifies whether the randomness in this
+            vmap should be the same or different across batches. If 'different',
+            the randomness for each batch will be different. If 'same', the
+            randomness will be the same across batches. If 'error', any calls to
+            random functions will error. Default: True.
```

Comment:
flag is called "randomness", not "use_batched_random"?

Reply:
Also, I'd add a comment somewhere that this differs from JAX's randomness behavior